trustpinning/certs_test.go

```diff
@@ -889,3 +784,11 @@ func generateExpiredTestingCertificate(rootKey data.PrivateKey, gun string) (*x5
 	startTime := time.Now().AddDate(-10, 0, 0)
 	return cryptoservice.GenerateCertificate(rootKey, gun, startTime, startTime.AddDate(1, 0, 0))
 }
+
+func generateRootKeyIDs(r *data.SignedRoot) {
```

Comment:
Non-blocking - a comment here to explain why this function is necessary (to generate data for an unexported cached field for equality testing) since I'm sure I will totally forget why this is here in a day or two :)